requirements.txt

```diff
@@ -1,5 +1,6 @@
 adafruit-circuitpython-rgb-display==3.11.0
 aiofiles==23.1.0
+argon2-cffi==21.3.0
```

Comment:
This is worth testing on a vanilla AmpliPi, as my setup is not very close to upstream at the moment.

Reply:
I'm gonna test this now with a fresh 0.2.1.

Reply:
she tested :nail_care: